src/smt/theory_str_noodler/theory_str_noodler.cpp

```diff
@@ -191,14 +191,17 @@ namespace smt::noodler {
         }
         // we need to propagate not(prefix) and not(suffix) before the actual solve (in init_search), because we need to ensure these axioms are 
```

Comment:
I would update the comment, it only talks about the negated versions and only about suffix/prefix.

Reply:
done